opentracing-api/src/main/java/io/opentracing/References.java

```diff
@@ -32,4 +32,9 @@ private References(){}
      * See http://opentracing.io/spec/#causal-span-references for more information about FOLLOWS_FROM references
      */
     public static final String FOLLOWS_FROM = "follows_from";
+
+    /**
+     * The self reference class can be used to construct a {@link Span} instance with a specific {@link SpanContext}.
+     */
+    public static final String SELF = "self";
```

Comment:
@tedsuo should this go through Specification RFC first?

If we are making this a new required reference type, there needs to be more details about the expected tracer behavior and/or description of the use case. For example, what happens to `startTime` field in this case?

Reply:
Sorry, just seeing this! Yes, @felixbarny an RFC would be appreciated, you can find the template here: https://github.com/opentracing/specification/blob/master/rfc_template.md